pkg/config/polling_manager_test.go

```diff
@@ -268,6 +302,77 @@ func TestNewPollingProjectConfigManagerOnDecision(t *testing.T) {
 	assert.Nil(t, err)
 }
 
+func TestNewPollingProjectConfigManagerHardcodedDatafile(t *testing.T) {
+	mockDatafile1 := []byte(`{"revision":"42"}`)
+	mockDatafile2 := []byte(`{"revision":"43"}`)
+	sdkKey := "test_sdk_key"
+
+	mockRequester := new(MockRequester)
```

Comment:
This `mockRequester` isn't passed as an option to `NewPollingProjectConfigManager` via `WithRequester`, so the assertion on 316 isn't useful, right?

Reply:
Thanks @mjc1283 for pointing out, i have added requester there.